lib/msal-angular/README.md

```diff
@@ -161,7 +161,7 @@ Defaults to `window.location.href`.
 
 * **validateAuthority** : Validate the issuer of tokens. Default is true.
 
-* **cacheLocation** : Sets browser storage to either 'localStorage' or sessionStorage'. Defaults is 'sessionStorage'.
+* **cacheLocation** : Sets browser storage to either 'localStorage' or 'sessionStorage'. Defaults is 'sessionStorage'.
```

Comment:
You can also make these backticks so they get formatted nicely:

```md
* **cacheLocation** : Sets browser storage to either `localStorage` or `sessionStorage`. Defaults is `sessionStorage`.
```
* **cacheLocation** : Sets browser storage to either `localStorage` or `sessionStorage`. Defaults is `sessionStorage`.

Reply:
Defaults **to** sessionStorage